### DISCUSSION TITLE
[ FF7 ] Crashes when Aerith death fmv is supposed to start playing.

### DISCUSSION THREAD

**Discussion by zerocoregithub** [julianxhokaxhiu/FFNx#505] [Q&A]
**Describe the bug**
FF7 crashes right when the Aerith death scene fmv is supposed to appear.
This happens on both a Windows 10 PC, as well as on a Steam Deck running SteamOS.
Tried using no mods, as well as every GPU renderer, but they all cause the same error.

**Screenshots**
<img width="639" alt="Capture" src="https://user-images.githubusercontent.com/114881049/193473905-338d996a-6cf9-4f0c-b8d5-0bd74d85fbfb.PNG">

**GPU**

- Brand: AMD Radeon Pro 5700 XT
- Driver Version: 22.9.1

[FFNx.log](https://github.com/julianxhokaxhiu/FFNx/files/9693652/FFNx.log)

**Comment by julianxhokaxhiu** [comment]
Please upload the `FFNx.log` file instead of copy-pasting it. Thank you.

**Reply by zerocoregithub** [reply]
My apologies. I've attached the log file to this message.

[FFNx.log](https://github.com/julianxhokaxhiu/FFNx/files/9693652/FFNx.log)

**Reply by julianxhokaxhiu** [reply]
Let's continue on Discord as you started the same thread there.
